guides/src/content/developer/tutorials/getting_started_tutorial.md

```diff
@@ -67,12 +67,13 @@ After you create the store application, switch to its folder to continue work di
 cd mystore
```

Comment:
@PSzyszkaSpark let's not add the note about SQLite - most people will use Postgres/MySQL and this causes some confussion